lib/virt_autotest/utils.pm

```diff
@@ -326,12 +326,12 @@ sub download_script {
 
     my $cmd = "curl -o ~/$script_name $script_url";
     $cmd = "ssh root\@$machine " . "\"$cmd\"" if ($machine ne 'localhost');
-    unless (script_retry($cmd, retry => 2, die => 0) == 0) {
+    unless (script_retry($cmd, timeout => 900, retry => 2, die => 0) == 0) {
```

Comment:
It is acceptable to enlarge the timeout for large size script. However, with time scale being 3, this will actually have time out as 2700 seconds. Will it be too large?

Reply:
Actually timeout is not always to be reached in test. In most cases commands will return true or false when they finish. Only when the command is running or it is hung, timeout is useful.

For large size vgpu drivers, 300M, it usually takes 6-10 minutes. if we keep the timeout 15 min, 15*60/3=300s, so how about decrease it to 300?

Reply:
Much better now. Thanks.